icicle/appUtils/ntt/tests/verification.cu

```diff
@@ -12,8 +12,14 @@
 #include "ntt/ntt_impl.cuh"
 #include <memory>
 
+#ifdef ECNTT_DEFINED
```

Comment:
note that since you don't compile mixed-radix ECNTT, then I think you should remove it from here and the makefile too since it will compare radix-2 ECNTT to itself which is not very useful

Reply:
actually with devmode - the mixed radix ecntt compilation time becomes feasible - so we may reintroduce it due to performance reasons